.circleci/config.yml

```diff
@@ -0,0 +1,62 @@
+version: 2.1
+
+workflows:
+  ruby-tests:
+    jobs:
+      - unit-tests:
+          matrix:
+            parameters:
+              version: ["2", "2.1", "2.2", "2.3", "2.4", "2.5", "2.6.5", "2.7"]
```

Comment:
What about "2.0" instead of "2" and then "2.6" instead of "2.6.5" to match the minor + minor?

Reply:
yep, good point, updating